### PR TITLE
Python: Add support for inline test expectations library

### PR DESCRIPTION
**Pull request by tausbn**
Adds `TestUtilities/InlineExpectationsTest.qll`, using the scheme introduced in #4137. Once this has been done, we can keep these (and future) copies synchronised in the usual way.

### CODE REVIEW

File: python/ql/test/TestUtilities/InlineExpectationsTest.qll [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+ * class ConstantValueTest extends InlineExpectationsTest {
+ *   ConstantValueTest() { this = "ConstantValueTest" }
+ *
+ *   override string getARelevantTag() {
+ *     // We only use one tag for this test.
+ *     result = "const"
+ *   }
+ *
+ *   override predicate hasActualResult(
+ *     Location location, string element, string tag, string valuesasas
```
Comment (RasmusWL) [inline python/ql/test/TestUtilities/InlineExpectationsTest.qll:31]: Is this one really supposed to be `valuesasas` and not `value`?

**Review by RasmusWL — APPROVED** [review]
Overall looks good to me, just one small thing I was surprised about.

File: python/ql/test/TestUtilities/InlineExpectationsTest.qll (hunk 1 above)
Comment (tausbn) [reply python/ql/test/TestUtilities/InlineExpectationsTest.qll:31]: You know, I stared at that a bit myself, trying to deduce its meaning... Perhaps @dbartol can enlighten us?

Comment (dbartol) [reply python/ql/test/TestUtilities/InlineExpectationsTest.qll:31]: Whoah, I don't know what happened there. I'm sure I intended it to be `value`.

Comment (tausbn) [reply python/ql/test/TestUtilities/InlineExpectationsTest.qll:31]: I can fix this up when I add these files to the list of identical files that should be kept synchronised. (I would need to make further changes to the CPP files anyway, as I think I may have removed some superfluous whitespace in the Python version.)

**Pull request merged**